### ISSUE TITLE
Bug Delete

### ISSUE THREAD

**Issue by Giaretta** [badernageral/bgfinancas#9]
Hi,

I'm having a problem with version 3.1.1. I can't delete any category or item, anything. I love this program and help me a lot. It is so useful and I dont want to change of software.

Thx

**Comment by badernageral** [comment]
Hi @Giaretta, sorry for the delay to respond, it's because I changed city recently, do not know if you're aware, but what may be happening is the following:
1 - you register a category
2 - you register an item
3 - you throw an expense / revenue / transfer using the item that belongs to a category
4 - **you can not delete the item if you launched it**
5 - **you can not delete a category if it has items**
In short, I believe that you launched the items, so you can not exclude them, to exclude should first remove the releases.
This restriction was made so that you do not delete items by accident.
